index.html

```diff
@@ -153,6 +153,25 @@ <h3><code>position="bottom left, bottom beforeright"</code> (fallback positions
   <div slot="handle">Expand ▶</div>
   <div slot="expandable">Expanded content</div>
 </juicy-popover>
+    
+    
+<h3><code>position="beforetop center"</code></h3>
+<juicy-popover id="rightexpand" position="beforetop center">
+    <div slot="handle">Expand ▶</div>
+    <div slot="expandable">Expanded content</div>
+</juicy-popover>
+ 
+<h3><code>position="middle center"</code></h3>
+<juicy-popover id="rightexpand" position="middle center">
+    <div slot="handle">Expand ▶</div>
+    <div slot="expandable">Expanded content</div>
+</juicy-popover>
+ 
+<h3><code>position="bottom center"</code></h3>
```

Comment:
Works great, but can you please move the demo of `bottom center` after the demo of `bottom left`?